backend/server/routers/programs.py

```diff
@@ -145,13 +145,15 @@ def getMinors(programCode: str):
     return {"minors": minrs}
 
 def convertSubgroupObjectToCoursesDict(object: str, description: str|list[str]) -> dict[str, str]:
-    """ Gets a subgroup object (format laid out in the processor) and fetches the exact courses its referring to """
+    """
+    Gets a subgroup object (format laid out in the processor) and
+    fetches the exact courses its referring to
+    """
     if " or " in object:
         return {c: description[index] for index, c in enumerate(object.split(" or "))}
-    elif not re.match(r"[A-Z]{4}[0-9]{4}", object):
+    if not re.match(r"[A-Z]{4}[0-9]{4}", object):
         return regex_search(object)
-    else:
-        return {object: description}
+    return {object: description}
```

Comment:
since this is just consecutive returns, how would you feel about a mutli line ternary like so:
```py
return return {c: description[index] for index, c in enumerate(object.split(" or "))} if "or" in object
           else regex_search(object) if not not re.match(r"[A-Z]{4}[0-9]{4}", object)
           else {object: description}
```

Reply:
yeah it's not too bad like that
~~if only we were using rust~~